src/Microsoft.Health.Fhir.Tests.Common/TestFiles/Normative/SearchParameter-Resource-id.json

```diff
@@ -0,0 +1,42 @@
+{
+    "resourceType": "SearchParameter",
+    "id": "Resource-idfoo",
+    "extension": [
+        {
+            "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status",
+            "valueCode": "trial-use"
+        }
+    ],
+    "url": "http://hl7.org/fhir/SearchParameter/Resource-idfoo",
```

Comment:
>idfoo [](http://example.com/codeflow?start=57&length=5)

I would change name of file to `idfoo` as well.

Reply:
Agreed, updated.